src/main/scala/nl/knaw/dans/easy/stage/Settings.scala

```diff
@@ -47,8 +47,6 @@ object Settings {
              urn: String,
              doi: String,
              otherAccessDoi: Boolean,
-             stageFileDataAsRedirectDatastreams: Boolean,
-             fileDataRedirectBaseUrl: Option[URL],
```

Comment:
The backward compatible call lacks the new argument. A [solution](https://github.com/DANS-KNAW-jp/easy-ingest-flow/blob/aca78d3a2cec0eb294a0c4453625a1c6ff764095/src/main/scala/nl.knaw.dans.easy.ingestflow/flowsteps/fedora/FlowStepCreateFedoraDataset.scala#L86-L108) (no PR yet) that requires way too much objects from the easy-stage-dataset component:
* the run method of the main class
* de settings object
* an exception class
* the disciplines loaded from fedora

I think we need something better than such a hack-ish solution.